docs/api-docs/webhooks/webhook-events.mdx

```diff
@@ -439,10 +439,65 @@ Payload objects with the following scopes take the form that follows:
   "producer": "stores/{store_hash}"
 }
 ```
+
+## Modifiers 
+
+| Name / Scope             | Description |
+|:-------------------------|:------------|
+| store/modifier/updated | Fires when you edit attributes for a local or shared modifier. Updates to the global store and channel locale as overrides triggers the webhook. The `context` fields are present for only override updates.  |
+
+The payload object takes the form that follows:
+
+```json filename="Example modifier payload object" showLineNumbers copy
+{
+  "scope": "store/modifier/updated",
+  "store_id": "1025646",
+  "data": {
+    "type": "shared_modifier",
+    "id": 205, // ID of the modifier
+    "affected_product_ids": [1, 2],
+    "context": {
+      "channel_id": 2,
+      "locale": "fr"
+    } 
+  },
+  "hash": "a833a57fadd56a32dc752fb6ca0841dc9602a495",
+  "created_at": 1561479233,
+  "producer": "stores/{store_hash}"
+}
+```
+
 ## Notifications
 
 Consult the [notifications section of the Channel Webhooks Guide](/docs/integrations/webhooks/events/channels#notifications).
 
+## Options
+
+| Name / Scope             | Description |
+|:-------------------------|:------------|
+| store/option/updated | Fires when you edit attributes for a local or shared variant option. Updates to the global store and channel locale as overrides triggers the webhook. The `context` fields are present for only override updates. |
```

Comment:
What are the attributes that trigger this webhook?

Reply:
updating variant option:
- display_name
- option_values